charts/dslorchestrator/templates/cronjob.yaml

```diff
@@ -13,7 +13,7 @@ spec:
   schedule: {{ .Values.crontime }}
   jobTemplate:
     spec:
-      backoffLimit: 2 # this has very low chance of failing, as all this does
+      backoffLimit: 0 # this has very low chance of failing, as all this does
```

Comment:
Please templatize as you have done for other jobs

Reply:
done